internal/output/format.go

```diff
@@ -30,6 +30,16 @@ func formatCost(currency string, d *decimal.Decimal) string {
 	return formatRoundedDecimalCurrency(currency, *d)
 }
 
+func formatUsageCost(out Root, cost *decimal.Decimal) string {
+	noUsageCosts := cost == nil || cost.Equals(decimal.Zero)
+
+	if usageCostsEnabled(out) || noUsageCosts {
+		return formatCost(out.Currency, cost)
```

Comment:
I think no usage costs check is backwards right? We show the cost if usageCostsEnabled or cost is non-zero


```suggestion
	hasUsageCosts := cost != nil && !cost.Equals(decimal.Zero)

	if usageCostsEnabled(out) || hasUsageCosts {
		return formatCost(out.Currency, cost)
```

Reply:
Oops, of course 👍